flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1341,26 +1365,26 @@ def test_created_is_listed(self):
 
     def test_listed_volume_attributes(self):
         """
-        ``list_volumes`` returns ``BlockDeviceVolume`` s that have the same
-        dataset_id and size as was passed to ``create_volume``.
+        ``list_volumes`` returns ``BlockDeviceVolume`` s that have the
+        same dataset_id and (maybe over-allocated size) as was passed
+        to ``create_volume``.
         """
         expected_dataset_id = uuid4()
+        requested_size = REALISTIC_BLOCKDEVICE_SIZE
```

Comment:
That's... confusing. Is there some expectation of the relationship between REALISTIC_BLOCKDEVICE_SIZE and allocation_unit()? seems like you can just use allocation_unit? E.g. int(allocation_unit() * 1.5) or something?